google/cloud/spanner_dbapi/connection.py

```diff
@@ -565,6 +576,24 @@ def run_partitioned_query(
             partitioned_query, statement.params, statement.param_types
         )
 
+    @check_not_closed
+    def set_autocommit_dml_mode(
```

Comment:
I think that it would make more sense to have a public method that takes a `AutocommitDmlMode` enum value as an argument, rather than one that takes a parsed statement as an argument. This method looks like a private method that should only be called by our own parser.

Instead, we should:
1. Have a private method that is basically the same as this.
2. Have a public method that takes an `AutocommitDmlMode` enum as an input argument and that actually changes the value of the flag. That method would then also be usable for anyone using the dbapi driver programmatically, as they could just call that method to change the value.

See https://github.com/googleapis/java-spanner-jdbc/blob/910a130a02f72c4d8764f12e347f6c3d1bd51b2b/src/main/java/com/google/cloud/spanner/jdbc/CloudSpannerJdbcConnection.java#L136 for how the API in the JDBC driver looks.

Reply:
Done